Bylaws.md

```diff
@@ -196,7 +204,7 @@ Members have discretion about which mailing list they use, but are strongly enco
 
 Members are strongly discouraged from posting the text of Member Mail List messages to the Forum Public Mail List without the permission of the author or commenter.
 
-These activities should be followed by any Chartered Working Group. Working Groups may use the Forum Member List but they are encouraged to maintain a separate Member Mail List. Working Groups may use the Forum Member Web Site but they must maintain a separate section designated for material related to the specific Working Group.
+These activities should be followed by any Chartered Working Group ensuring the use of the Working Group's Public Mail List instead of the Forum-level Public Mail List. Working Groups may use the Forum Member List but they are encouraged to maintain a separate Member Mail List. Working Groups may use the Forum Member Web Site but they must maintain a separate section designated for material related to the specific Working Group.
```

Comment:
I'm not sure about the intention of `These activities should be followed by any Chartered Working Group` here.

The first sentence states `ensure the use`, while the second sentence continues with `may use`, what is the purpose of this addition?

```suggestion
Chartered Working Groups should use the Working Group's Public Mail List instead of the Forum-level Public Mail List when such list is availible. Working Groups may use the Forum Member List but they are encouraged to maintain a separate Member Mail List. Working Groups may use the Forum Member Web Site but they must maintain a separate section designated for material related to the specific Working Group.
```

Reply:
The intent is to force each CWG to use its own public mailing list and not the Forum's public list for their CWG activities. See https://docs.google.com/document/d/1EtrIy3F5cPge0_M-C8J6fe72KcVI8H5Q_2S6S31ynU0/edit#heading=h.my9hanmafjby for more information. 

It is very confusing when the public expects to find minutes or other Bylaws-mandated posts to the CWG's public list and the Chair/Vice-Chair decides to post using the Forum's public list. The "may" was probably left from the time of the governance reform. I don't think this is needed anymore. 

Is it ok to keep the previous language?

Reply:
The problem is that the current/updated language doesn't read well. Do you suggest deleting the sentence: `Working Groups may use the Forum Member List but they are encouraged to maintain a separate Member Mail List.`?

That would resolve the conflicting language, but the first sentence would still require a re-write. 

Combined this would make:

```suggestion
These activities should be followed by any Chartered Working Group.  Chartered Working Groups shall use the Working Group's Mail Lists instead of the Forum-level Mail Lists. Working Groups may use the Forum Member Web Site but they must maintain a separate section designated for material related to the specific Working Group.
```

I'm not convinced that the first part of this sentence is adding any value.